coverage/.resultset.json

```diff
@@ -11,29 +11,32 @@
       "/Users/charlesellison/turing/3module/projects/the_pivot/app/models/cart.rb": [
         1,
         1,
+<<<<<<< HEAD
```

Comment:
Looks like there's a merge conflict in here that hasn't been resolved. Recommend you run rspec, which I think will just overwrite this file.